src/components/Personalization/turbine/initRuleComponentModules.js

```diff
@@ -15,18 +15,21 @@ import createSetHtml from "../actions/setHtml";
 import createSetText from "../actions/setText";
 import createSetAttribute from "../actions/setAttribute";
 import createSetImageSource from "../actions/setImageSource";
+import createResize from "../actions/resize";
 
 export default collect => {
   const setHtml = createSetHtml(collect);
   const setText = createSetText(collect);
   const setAttribute = createSetAttribute(collect);
   const setImageSource = createSetImageSource(collect);
+  const resize = createResize(collect);
 
   return {
     elementExists,
     setHtml,
     setText,
     setAttribute,
-    setImageSource
+    setImageSource,
+    resize
```

Comment:
It seems like the `move`, `resize`, and `setStyle` actions are basically identical (except that `setStyle` supports `priority` whereas the other two don't). Rather than create an action module for each, you could map to the same module here if you wanted to. Something like:

```
return {
  ...,
  setStyle,
  resize: setStyle,
  move: setStyle
}
```

If you want each of them to have their own module though just so we retain that one-to-one mapping, that's fine with me too.

Reply:
Yeah, I would like to retain the one-to-one mapping so we could add logging and tracing and other details later.